distmv/normal.go

```diff
@@ -354,3 +355,31 @@ func (n *Normal) setSigma() {
 		n.sigma.FromCholesky(&n.chol)
 	})
 }
+
+// TransformNormal transforms the vector src, generated from a multidimensional
+// standard normal, to one generated under the distribution specified by the
+// receiver. If dst is non-nil, the result will be stored into dst, otherwise
+// a new slice will be allocated. TransformNormal will panic if the length of
+// src is not the dimension of the Normal, or if dst is non-nil and len(src) !=
+// len(dst).
+func (n *Normal) TransformNormal(dst, src []float64) []float64 {
```

Comment:
That's not emphasised by the docs. I think it could be clearer. In that case, still method, but with the same name or TransformNorm, and the uniform source case TransformUnif to be explicit.